ci-operator/config/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-master.yaml

```
@@ -129,6 +137,27 @@ tests:
       GATHER_CAPI_LOGS: "true"
       TEST_SUITE: full
     workflow: assisted-baremetal
+- as: e2e-assisted-operator-capi-periodic
```

there's a new naming convension:
```suggestion
- as: e2e-ai-operator-ztp-capi-periodic
```

---

Why ztp?
Does it have to have ztp in the name?

---

We picked that annotation to include any flow that doesn't rely on the user fetching the ISO and booting it himself outside of CRs
Is it not the case for capi operator flow?